ambry-messageformat/src/main/java/com.github.ambry.messageformat/MessageFormatRecord.java

```diff
@@ -257,6 +264,14 @@ public static BlobAll deserializeBlobAll(InputStream stream, StoreKeyFactory sto
         headerBuf.rewind();
         header = new MessageHeader_Format_V2(headerBuf);
         break;
+      case Message_Header_Version_V3:
```

Comment:
it seems no test covers this piece of code.

Reply:
added

Reply:
Actually, this piece of code should be tested out in `MessageFormatInputStreamTest.java`, you can take a look at `messageFormatRecordsTest` in that file.